xbmc/addons/kodi-dev-kit/include/kodi/addon-instance/pvr/Recordings.h

```diff
@@ -492,6 +497,49 @@ class PVRRecording : public CStructHdl<PVRRecording, PVR_RECORDING>
   /// @brief To get with @ref SetProviderName changed values.
   std::string GetProviderName() const { return m_cStructure->strProviderName; }
 
+  /// @brief **optional**\n
+  /// Age rating for the recording.
+  void SetParentalRating(unsigned int iParentalRating)
+  {
+    m_cStructure->iParentalRating = iParentalRating;
+  }
+
+  /// @brief To get with @ref SetParentalRating changed values
+  unsigned int GetParentalRating() const { return m_cStructure->iParentalRating; }
+
+  /// @brief **optional**\n
+  /// Parental rating code for this recording.
+  void SetParentalRatingCode(const std::string& ratingCode)
+  {
+    strncpy(m_cStructure->strParentalRatingCode, ratingCode.c_str(),
+            sizeof(m_cStructure->strParentalRatingCode) - 1);
+  }
+
+  /// @brief To get with @ref SetParentalRatingCode changed values.
+  std::string GetParentalRatingCode() const { return m_cStructure->strParentalRatingCode; }
+
+  /// @brief **optional**\n
+  /// Parental rating icon URL for this recording.
+  void SetParentalRatingIcon(const std::string& ratingIcon)
+  {
+    strncpy(m_cStructure->strParentalRatingIcon, ratingIcon.c_str(),
+            sizeof(m_cStructure->strParentalRatingIcon) - 1);
+  }
+
+  /// @brief To get with @ref SetParentalRatingIcon changed values.
+  std::string GetParentalRatingIcon() const { return m_cStructure->strParentalRatingIcon; }
+
+  /// @brief **optional**\n
+  /// The source that the backend used to determine the event's parental rating.
```

Comment:
=> The source used to determine the recording's parental rating.

Please also note change from event's to recording's.